src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -89,6 +93,12 @@ public AutoResetEvent RestartEvent
             }
         }
 
+        public static void ResetStandbyMode()
+        {
+            // this is for testing only
+            _standbyMode = null;
+        }
+
```

Comment:
Moved this because StyleCop was complaining about the order (properties before methods)